testing/ponysay/APKBUILD

```diff
@@ -0,0 +1,22 @@
+# Maintainer: Matthew T Hoare <matthew.t.hoare@gmail.com>
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Erkin Batu Altunbaş <erkinbatu@gmail.com>
+
+pkgname=ponysay
+pkgver=3.0.2
+pkgrel=0
+pkgdesc="cowsay reimplemention for ponies"
+url="http://erkin.github.com/ponysay/"
+arch="all"
```

Comment:
Are there arch specific files in here, otherwise it should be noarch.

Reply:
Thnaks!